CDP4Dal.Tests/Operations/PostOperationTestFixture.cs

```diff
@@ -48,13 +50,5 @@ public void VerifyThatConstructorSetsLists()
 
     internal class TestPostOperation : PostOperation

```

Comment:
is this class still required?

Reply:
Yes, because PostOperation is an abstract class

Reply:
PostOperation does not know about Json Serialization, so it does not have JsonPropertyname attributes set. Therefore this specific TestPostOperation should stay.